### PR TITLE
CRUD and seed station

### PR DESCRIPTION
**Pull request by mimizotti**
@s-espinosa, @dphilla, @snayrouz, @amhursh:

Created methods for the following: controllers, models, and views.
Seeded `station.csv`

### CODE REVIEW

**Comment by snayrouz** [comment]
@amhursh we finished iteration 1 and 2 tonight. I'll catch you up tomorrow at school on everything if you have any questions. The seeding got interesting but we got it to pass.

File: app/controllers/bike_share_app.rb
Comment (s-espinosa) [inline app/controllers/bike_share_app.rb]: Can also do this in a single call:

```ruby
Station.destroy(params[:id])
```

Also, don't need to be saving this to an instance variable since you're not using it in a view that you're rendering.

File: db/seeds.rb
Comment (s-espinosa) [inline db/seeds.rb:7]: Looks like this is doing what you need it to, but check into [::foreach](http://ruby-doc.org/stdlib-2.0.0/libdoc/csv/rdoc/CSV.html#method-c-foreach). Will be able to accomplish the same task with fewer steps.

File: spec/features/user_can_access_new station_page_spec.rb
Comment (s-espinosa) [inline spec/features/user_can_access_new station_page_spec.rb]: Should work if you remove the trailing `/`.

File: spec/features/user_can_delete_stations_spec.rb
Comment (s-espinosa) [inline spec/features/user_can_delete_stations_spec.rb:11]: Would like some more assertions here. You've gone through the process of creating two stations. Would recommend asserting that you still see the one that you didn't delete, and that you don't see the one that you did. You don't have to check every attribute; name would probably be enough.

File: spec/features/user_can_edit_existing_station_spec.rb
Comment (s-espinosa) [inline spec/features/user_can_edit_existing_station_spec.rb]: Maybe just `Update Station` for this button name.

**Review by s-espinosa — COMMENTED** [review]
@mimilettd @snayrouz @dphilla @amhursh Think this looks good! 🎉 Comments below. Let Ilana and I know if you have any questions.

File: app/controllers/bike_share_app.rb
Comment (amhursh) [reply app/controllers/bike_share_app.rb]: Agreed. Will update before merging.

File: db/seeds.rb
Comment (amhursh) [reply db/seeds.rb:7]: I agree. Less steps is good, and I think ::foreach is more of a standard csv parse that is easier for others to recognize and understand quickly. Will update prior to merge.

File: spec/features/user_can_access_new station_page_spec.rb
Comment (amhursh) [reply spec/features/user_can_access_new station_page_spec.rb]: Agreed, Will do.

**Comment by snayrouz** [comment]
Let's wait until we're together at Turing this morning to merge. I'd like to talk these changes through as a group so we all understand them. I'll assign a card to Mimi to refactor that method since she so kindly volunteered. I'll be at school in 30 minutes

File: spec/features/user_can_delete_stations_spec.rb
Comment (amhursh) [reply spec/features/user_can_delete_stations_spec.rb:11]: I also like the idea of testing the non-deleted station is still there. Will add a few `has_content` assertions to make it more comprehensive.

**Review by amhursh — CHANGES_REQUESTED** [review]
Looks good. We should be able to make a few changes (primarily that csv merge and some other refactoring) and merge once they are pushed.

**Comment by amhursh** [comment]
@mimilettd @snayrouz @dphilla I committed changes addressing all the comments from Sal except the CSV parse. I will hold on merging until I have company in here so I can explain.

**Review by amhursh — APPROVED** [review]

**Pull request merged**
